stake-pool/program/src/instruction.rs

```diff
@@ -1363,6 +1363,10 @@ pub fn decrease_additional_validator_stake_with_vote(
 
 /// Creates `UpdateValidatorListBalance` instruction (update validator stake
 /// account balances)
+#[deprecated(
+    since = "1.1.0",
```

Comment:
@joncinque @buffalojoec please update this vers number if not appropriate

Reply:
I think that's good, since we're not removing the deprecated helper just yet. Whenever we decide to do so, we can bump to `2.0.0`.